test/TestPrefers.cc

```diff
@@ -0,0 +1,31 @@
+#include <Vertex.h>
+#include <GraphReader.h>
+#include <CriticalRSM.h>
+#include <Utils.h>
+#include "TestDefs.h"
+
+TEST_CASE("Helper method of CriticalRSM", "[matching_critical_rsm]") {
+    std::shared_ptr<Vertex> a1 = std::make_shared<Vertex>("a1");
+    std::shared_ptr<Vertex> b1 = std::make_shared<Vertex>("b1");
+    std::shared_ptr<Vertex> b2 = std::make_shared<Vertex>("b2");
+    std::shared_ptr<Vertex> b3 = std::make_shared<Vertex>("b3");
+
+    PreferenceList a1_pref_list;
+    a1->get_preference_list().emplace_back(b1);
+    a1->get_preference_list().set_tie(1, b2);
+    a1->get_preference_list().set_tie(1, b3);
+
+    auto M = Matching();
+    auto G = read_graph(get_filepath(get_resources_dir(), "/weak_stable.txt")); // required only for critical RSM constructor, doesn't affect output of the function
```

Comment:
Put this comment before the code in its own line.
Its not completely necessary, but wrap the lines to 80 column width.